packages/web/src/pages/search-page-v2/SearchFilters.tsx

```diff
@@ -68,11 +68,19 @@ const MoodFilter = () => {
       marginBottom: 0
     }
   }
+  const moodLabelCss = {
+    '& .emoji': {
```

Comment:
yeah interesting, we should def reach out to design and get emojiis in a cleaner state and in harmony. writing a note